nix/pkgs/mls-test-cli/default.nix

```diff
@@ -19,9 +17,16 @@ rustPlatform.buildRustPackage rec {
     sha256 = "sha256-/XQ/9oQTPkRqgMzDGRm+Oh9jgkdeDM1vRJ6/wEf2+bY=";
     rev = "c6f80be2839ac1ed2894e96044541d1c3cf6ecdf";
   };
+  cargoLock = {
+    lockFile = "${src}/Cargo.lock";
+    outputHashes = {
+      "openmls-0.4.1" = "sha256-oEPziXyGmPV6C80lQpi0z7Ehl3/mGFz0HaePT8h3y0Q=";
+      "ring-0.17.0-not-released-yet" = "sha256-n8KuJRcOdMduPTjDBU1n1eec3w9Eat/8czvGRTGbqsI=";
+      "x509-parser-0.13.1" = "sha256-ipHZm3MmiOssGkFC5O4h/Y3p1U0aj7wu+LGaBuQImuU=";
+    };
+  };
   doCheck = false;
-  cargoSha256 = "sha256-AlZrxa7f5JwxxrzFBgeFSaYU6QttsUpfLYfq1HzsdbE=";
-  cargoDepsHook = ''
-    mkdir -p mls-test-cli-${version}-vendor.tar.gz/ring/.git
+  preBuild = ''
+    mkdir $CARGO_HOME/cargo-vendor-dir/ring-0.17.0-not-released-yet/.git
```

Comment:
Could you please add a comment explaining this?

Reply:
Thank you, I have done so.